lib/bucketed_game.nit

```diff
@@ -32,6 +35,8 @@ end
 # Something acting on the game from time to time
 class Bucketable[G: Game]
 	super Turnable[G]
+	auto_serializable
```

Comment:
Yup, it adds serialization of the local variables (`act_at`) and allow to deserialize instances of this precise class.